### PR TITLE
Fix #4: Added sensor initialization for 461x boards and Fix #6: updated README.md

### LINKED ISSUE #4: Initialize BME280 and Si1133 to determine the low power consumption
Initialize the sensors BME 280 and Si1133 in the sketch "stm32l0_pwrtest" and put them in sleep mode to determine the power consumption.

### LINKED ISSUE #6: Update the README.md with power consumption graph for 4612 and 4801
Update README with power consumption graph. As of now for 4612 and 4801.
Also mention the key things to do to achieve low power consumption.

### PR DESCRIPTION
**Pull request by dhineshkumarmcci**
Added sensor initialization module for Catena 4610, 4611 and 4612 boards.

### CODE REVIEW

File: stm32l0_pwrtest.ino [hunk 1]
```diff
@@ -183,6 +193,15 @@ void setup_platform()
         // set up flash
         setup_flash();
 
+#ifdef ARDUINO_MCCI_CATENA_4610 || ARDUINO_MCCI_CATENA_4611 || \
+	\ ARDUINO_MCCI_CATENA_4612
```
Comment (svelmurugan92) [inline stm32l0_pwrtest.ino:219]: Make sure booster is turn off for 4612.

**Comment by svelmurugan92** [comment]
Have to update the version

**Comment by svelmurugan92** [comment]
Please, Print the system clock frequency and possibly VDD voltage
https://github.com/mcci-catena/stm32l0_pwrtest/blob/9f02385df4594c52755a7d762f0d7f0a7af05902/stm32l0_pwrtest.ino#L184-L191

File: stm32l0_pwrtest.ino (hunk 1 above)
Comment (dhineshkumarmcci) [reply stm32l0_pwrtest.ino:219]: Added the booster part for 4612

**Comment by dhineshkumarmcci** [comment]
Updated the sketch with suggested changes.

**Comment by svelmurugan92** [comment]
It looks good. @terrillmoore @chwon64  Shall we merge this pull request?

**Comment by dhineshkumarmcci** [comment]
Have updated the README.md with power consumption of graph of Catena 4612 and Catena 4801. #6

**Review by chwon64 — APPROVED** [review]

**Pull request merged**